optimum/exporters/onnx/convert.py

```diff
@@ -573,24 +576,35 @@ def remap(value):
                 dynamic_axes=dynamix_axes,
                 do_constant_folding=do_constant_folding,
                 opset_version=opset,
+                dynamo=use_dynamo,
+                verify=verify_accuracy,
+                artifacts_dir = Path(output.as_posix()).parent,
+                report=debug_reports,
+                dump_exported_program=debug_reports,
```

Comment:
what's the min torch version for these apis ?

Reply:
then let's either guard/dispatch based on version or group these in a dynamo_kwargs dict that's populated based on torch version.